private/buf/bufwire/image_reader.go

```diff
@@ -97,31 +97,11 @@ func (i *imageReader) GetImage(
 		}
 		span.End()
 	case buffetch.ImageEncodingJSON:
-		firstProtoImage := &imagev1.Image{}
-		_, span := i.tracer.Start(ctx, "first_json_unmarshal")
-		if err := protoencoding.NewJSONUnmarshaler(nil).Unmarshal(data, firstProtoImage); err != nil {
-			span.RecordError(err)
-			span.SetStatus(codes.Error, err.Error())
-			span.End()
-			return nil, fmt.Errorf("could not unmarshal image: %v", err)
-		}
-		// TODO right now, NewResolver sets AllowUnresolvable to true all the time
-		// we want to make this into a check, and we verify if we need this for the individual command
-		span.End()
-		_, newResolverSpan := i.tracer.Start(ctx, "new_resolver")
-		resolver, err := protoencoding.NewResolver(
-			bufimage.ProtoImageToFileDescriptors(
-				firstProtoImage,
-			)...,
-		)
+		resolver, err := i.boolstrapResolver(ctx, protoencoding.NewJSONUnmarshaler(nil), data)
```

Comment:
This still didn't get fixed, you changed from `boolStrap` to `boolstrap` in https://github.com/bufbuild/buf/pull/2293/commits/801d054d1bcda8f8dd30d3aa45c5b75b01d42de2. I'm going to fix myself